data/Building/comment.js

```diff
@@ -65,4 +72,51 @@ async function comment(buildingId, posterId, message) {
   ).exec();
 }
 
-module.exports = comment;
+/**
+ * Modify a comment
+ * @param {string} buildingId The id of the building to take the comment
+ * @param {String} posterId The poster's ID
+ * @param {String} oldMessage The old message to idenify which comment to change
+ * @param {String} newMessage The new message replacing the old message
+ */
+async function modifyComment(buildingId, posterId, oldMessage, newMessage) {
```

Comment:
This function assumes that the same message cannot be sent multiple times (even if by different people). This should be done by the message ID, instead, to avoid this issue.